src/main.cpp

```diff
@@ -0,0 +1,76 @@
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <pcap.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <iostream>
+
+void packet_handler(u_char *user_data, const struct pcap_pkthdr *pkthdr,
+                    const u_char *packet) {
+  std::cout << "Packet capture length: " << pkthdr->caplen << std::endl;
+  std::cout << "Packet total length: " << pkthdr->len << std::endl;
+}
+
+int main() {
+  char errbuf[PCAP_ERRBUF_SIZE];
+  pcap_if_t *alldevs;
+  pcap_if_t *d;
+  pcap_t *handle;
+  int index = 0;
+  int select_number = 0;
+
+  // 모든 네트워크 디바이스 찾기
+  if (pcap_findalldevs(&alldevs, errbuf) == -1) {
+    std::cerr << "Error finding devices: " << errbuf << std::endl;
+    return 1;
+  }
+
+  // 사용 가능한 디바이스 출력
+  for (d = alldevs; d != NULL; d = d->next) {
+    std::cout << ++index << ". " << d->name;
+    if (d->description)
+      std::cout << " (" << d->description << ")" << std::endl;
+    else
+      std::cout << " (No description available)" << std::endl;
+  }
+
+  // 첫 번째 디바이스 사용
+  if (alldevs == NULL) {
+    std::cerr << "No devices found" << std::endl;
+    return 1;
```

Comment:
```suggestion
    return EXIT_FAILURE
```
더 명시적인 것도 선택할 수 있어보입니다!

Reply:
수정 완료했습니다 ~~